client/python/modeldb/utils/ConfigUtils.py

```diff
@@ -1,68 +1,71 @@
 import sys
-import yaml
+import json
 import ConfigConstants as constants
 
 class ConfigReader(object):
     def __init__(self, filename):
         # TODO: need to deal with errors here
-        self.config =  yaml.load(file(filename, 'r'))
+        self.config =  json.load(file(filename, 'r'))
         self.validate_config()
 
     def validate_config(self):
         # TODO: what all do we expect to be here?
-        if constants.PROJECT_KEY not in self.config:
+        if 'project' not in self.config:
             print "Project must be defined in config file. Cannot proceed."
             sys.exit(-1)
-        if constants.GIT_USERNAME_KEY not in self.config:
```

Comment:
Ok got it. How about the following?
- let's remove constants.ABC; we can use the actual strings when reading in the config file
- when creating objects from the config (e.g. for project etc), let's define structs s.t. we can reference their fields directly. Then we don't need to use the keys in some places and not use them in others.

So we can have a similar interface for all accesses to the config.